keyboards/keebsforall/coarse60/readme.md

```diff
@@ -0,0 +1,16 @@
+# Coarse60
+### seh0nky x KeebsForAll 
+
+A beginner-friendly aluminum alice inspired keyboard. The Coarse60 aims to provide a seamless aluminum keyboard with premium features for an affordable price.
+
+![coarse60](https://i.imgur.com/L6dw2xKh.jpg)
+
+* Hardware Supported: seh0nky x KeebsForAll Coarse60
+* Keyboard Maintainer: [e11i0t23](https://github.com/e11i0t23)
+* Keyboard Availabilty: [keebsforall](https://keebsforall.com) 
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```